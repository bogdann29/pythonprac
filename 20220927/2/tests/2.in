8,2,89,3,2,100,835,2458,23,7,12,8,90,1,45,456,34,2,359
