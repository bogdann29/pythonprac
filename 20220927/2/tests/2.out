[100, 90, 1, 2, 2, 2, 3, 89, 45, 835, 23, 456, 12, 7, 34, 2458, 8, 8, 359]
